src/hotspot/cpu/aarch64/aarch64_sve_ad.m4

```diff
@@ -1922,3 +1922,5 @@ VECTOR_INSERT(B, iRegIorL2I, B, Register)
 VECTOR_INSERT(S, iRegIorL2I, H, Register)
 VECTOR_INSERT(I, iRegIorL2I, S, Register)
```

Comment:
I'm sorry that I didn't look at the implementation PR for integer type. Is it better to merge the rules for "B/H/S" as a single one?

Reply:
> I'm sorry that I didn't look at the implementation PR for integer type. Is it better to merge the rules for "B/H/S" as a single one?

Agree. I plan to do some cleanup work before integrating to jdk mainline, which will include such kind of merging. If @Wanghuang-Huawei can do it in this patch, that would help to reduce future refactoring work.

Reply:
It's a good idea. However, we can not comment the type info after merging, so I choose this implementation.